src/main/java/io/swagger/inflector/models/ApiError.java

```diff
@@ -16,10 +16,20 @@
 
 package io.swagger.inflector.models;
 
+import javax.ws.rs.core.Response;
+import java.util.concurrent.ThreadLocalRandom;
+
 public class ApiError {
     private int code;
     private String message;
 
+    public static ApiError createInternalError() {
+        final String message = String.format("There was an error processing your request."
```

Comment:
I would prefer if the Pojo stayed "dumb" and didn't have the `ThreadLocalRandom` import.  Perhaps we can pass the message into the `ApiError` class when calling `createInternalError` instead of having the generation in the pojo itself